AdminCore.DataETL/CsvChoEtlAdapter.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using AdminCore.Common.Attributes.DataEtl;
+using AdminCore.Common.Interfaces;
+using AutoMapper;
+using Castle.Core.Internal;
+using ChoETL;
+
+namespace AdminCore.DataETL
+{
+    public class CsvChoEtlAdapter : IDataEtlAdapter
```

Comment:
what's `cho`? could we come up with a more meaningful name for this class?

Reply:
ChoEtl is the name of the ETL library https://github.com/Cinchoo/ChoETL
I think the name refers to it well enough unless you think differently?

Reply:
I don't have a better idea

Reply:
I added an xml documentation comment to the class name describing purpose/implementation.
```
    /// <summary>
    /// Implements the ETL Library ChoEtl for conversion of data to csv.
    /// https://github.com/Cinchoo/ChoETL
    /// </summary>
```